src/components/FilterSizes.js

```diff
@@ -8,58 +8,100 @@ class FilterSizes extends React.Component {
     }
 
-    changeText = (text, key) => {
-        let arr = [false, false, false, false, false];
-        arr[key] = true;
-        this.setState({text: text, clicked: arr});
+    clickStart = () => {
+        this.setState({index: 0}, () => {
+            this.changeText();
+        });
+    }
+
+    clickNext = () => {
+        this.setState((prevState) => {
+            return {index: prevState.index +1};
+        }, () => {
+            this.changeText();
+        });
+    }
+
+    clickBack = () => {
+        this.setState((prevState) => {
+            return {index: prevState.index - 1};
+        }, () => {
+            this.changeText();
+        });
+    }
+
+    changeText = () => {
+        let selected = [false, false, false, false, false];
+        selected[this.state.index] = true;
+        this.setState({text: this.state.paragraphs[this.state.index], selected: selected});
     }
     
     render () {
-        let params = [ //text, style.top, style.left
-            [<p>Notice how our input, a 6 x 6 RGB image, is split into three 6 x 6 layers.</p>, "90%", "19%"],
-            [<p>This is the symbol for convolution. The filters <em>convolve</em> over the image.</p>, "54%", "34.75%"],
-            [<><p>
-                Filter 1 and Filter 2 both have three layers. The first layer of each filter
-                convolves over the first layer of the input image, the second layer of each filter convolves over the
-                second layer of the input image, and so on. Then, for each filter, the three resulting matrices are summed up to produce an output of one layer.
-                </p>
-                <p>In order for our outputs to be the same size, Filter 1 and Filter 2 must be the same size as each other.</p></>, "47%", "47%"],
-            [<><p>
-                Our 3 x 3 filters can only move along each row of the input image four times until it hits the right side (try working it out yourself!).
-                Similarly, it can only move down a row four times until it hits the bottom, giving our output (also known as a <em>feature map</em>) a size of 4 x 4.
-            </p>
-            <p>If we use filters of bigger sizes, our output matrix will be smaller, and will capture less information.
-                As a result, we should be careful when choosing our filter size.
-            </p></>, "47%", "61.5%"], 
-            [<><p>
-                Each filter produced an output of size 4 x 4 x 1. We then stack the result of each filter convolution, to produce an output of size 4 x 4 x 2.
-            </p>
-            <p>
-                This output now acts as the input to the next layer of our CNN!
-            </p> </>, "78%", "81%"]
+        let params = [ //style.top, style.left
+            ["90%", "19%"],
+            ["54%", "34.75%"],
+            ["47%", "47%"],
+            ["47%", "61.5%"], 
+            ["78%", "81%"]
         ];
         
         const buttons = params.map((val, index) => {
```

Comment:
These aren't buttons anymore, right? Do you mind changing the variable name for clarity then?